src/Chapter1.hs

```diff
@@ -641,7 +669,11 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | n < 0 = firstDigit (0 - n)
```

Comment:
Interesting approach :thinking: Though, I think you can use standard function `abs` :slightly_smiling_face: 
Also, you can see that `n` will be compared with zero on each recursive call, which is redundant, as it is enough to check only once. If you want to improve this solution, you can introduce a helper recursive function.